scripts/bench-hyperfine.sh

```diff
@@ -57,7 +57,6 @@ run_bench() {
         -L"$OUTPUT_DIR/" \
         -Wl,-rpath "$MLIR_DIR/lib" \
         -Wl,-rpath "$OUTPUT_DIR" \
-        -Wl,--rpath-link "$OUTPUT_DIR" \
```

Comment:
Out of curiosity, why is this argument unnecessary?

Reply:
it wasnt necessary, i added it somewhen trying some stuff, but this argument doesnt exist on macos